index.html

```diff
@@ -16,20 +16,26 @@
     <title>portfolio skeleton</title>
   
   <body>
-    <img src="images/Light.jpg" alt="main">
+    <img class="mobile" src="images/Light.jpg" alt="main">
         <header class="container">
           <div class="section">
             <div class="logo-holder">
-            <a href="#" class="my-logo"><img src="images/MyLogo.png " alt="my-logo"> </a>
+           <a href="#"><p>My logo</p></a>
             </div>
-          <nav class="hamburger">
-          <a href="#"><i class="fa fa-bars"></i></a>
-          <ul class="ul-style">
+            <ul class="ul-style">
+              <li>
+              <a class="portfolio" href="#">portfolio</a>
+            </li>
             <li>
-            <a class="portfolio" href="#"></a>
-          </li>
-          
-        </ul>       
+              <a class="frame3" href="#">contact</a>
+            </li>
+            <li>
+              <a class="frame5" href="#">about</a>
+            </li>
+           
+          </ul> 
```

Comment:
- [ ] The Item Navigation should be aligned to the right and the background color is not matching. Kindly fix it
### Figma design
![header-figma](https://user-images.githubusercontent.com/81727988/167232450-4b999efa-ca1c-457b-9e6c-9da93789dc74.png)
### your design
![his](https://user-images.githubusercontent.com/81727988/167232459-373b94c0-6e2d-479d-9890-1d880ac8c79c.png)